### PR TITLE
Upgrade to polkadot-1.3.0

### PR DESCRIPTION
**Pull request by Moliholy**

### CODE REVIEW

**Review by stiiifff — DISMISSED** [review]

File: node/src/rpc.rs [hunk 1]
```diff
@@ -41,7 +41,6 @@ where
 		+ Send
 		+ Sync
 		+ 'static,
-	C: CallApiAt<Block>,
```
Comment (weezy20) [inline node/src/rpc.rs:44]: What is the reason for getting rid of this?

Comment (Moliholy) [reply node/src/rpc.rs:44]: It's not a required constrain. For instance in the main [node's implementation](https://github.com/substrate-developer-hub/substrate-node-template/blob/main/node/src/rpc.rs) it is not included.

File: runtime/mainnet/src/lib.rs [hunk 2]
```diff
@@ -448,6 +451,22 @@ impl parachain_info::Config for Runtime {}
 
 impl cumulus_pallet_aura_ext::Config for Runtime {}
 
+parameter_types! {
+	/// The asset ID for the asset that we use to pay for message delivery fees.
+	pub FeeAssetId: AssetId = Concrete(xcm_config::RelayLocation::get());
```
Comment (weezy20) [inline runtime/mainnet/src/lib.rs:456]: Is this the correct understanding that for all parachains, where XCM delivery is facilitated by XCMP-Lite ( HRMP ), the `FeeAssetId` must be the Relay chain native asset?

Comment (Moliholy) [reply runtime/mainnet/src/lib.rs:456]: As far as I saw all XCM fees are always paid in the relay chain's native currency, regardless of being the actual relay chain or one of its parachains. For instance, in all AssetHub instances (as a particular case of a parachain) the fee is paid with DOT / KSM / ROC, etc. depending on the corresponding relay chain.

Comment (weezy20) [reply runtime/mainnet/src/lib.rs:456]: I have observed the same. The question stems more from the fact that if it's the only option given the message delivery is mediated by the relay-chain and not p2p between sibling chains. For now, to me it seems that this config item is generic to allow for payment by different `MultiAssets` once p2p xcmp is available.

**Review by weezy20 — APPROVED** [review]

**Pull request merged**